src/NexusMods.App.UI/Pages/ObservableInfo/ObservableInfoPageViewModel.cs

```diff
@@ -36,13 +52,52 @@ public ObservableInfoPageViewModel(IWindowManager windowManager) : base(windowMa
                         self._trackingStatesBuffer.Add(trackingState);
                     });
 
-                    self._trackingStatesBuffer.Sort();
+                    var counts = new Dictionary<string, int>(StringComparer.OrdinalIgnoreCase);
+
+                    foreach (var state in self._trackingStatesBuffer)
+                    {
+                        var count = counts.GetValueOrDefault(state.FormattedType, 0);
+                        counts[state.FormattedType] = count + 1;
+                    }
```

Comment:
Do this in the loop above to avoid iterating twice
```csharp
                    var counts = new Dictionary<string, int>(StringComparer.OrdinalIgnoreCase);
                    
                    ObservableTracker.CheckAndResetDirty();
                    ObservableTracker.ForEachActiveTask(trackingState =>
                    {
                        if (self._trackingStatesBuffer.Add(trackingState))
                        {
                            var count = counts.GetValueOrDefault(trackingState.FormattedType, 0);
                            counts[trackingState.FormattedType] = count + 1;
                        }
                        
                    });
```

Reply:
Not sure if you actually want the `if`, if you duplicates you might want to count those, but not sure